requirements.txt

```diff
@@ -0,0 +1,2 @@
+ -e .
```

Comment:
can we remove this file? there is a [noxfile.py](https://github.com/optimizely/kubeluigi/blob/master/noxfile.py) which is the standard for testing this kind of packages: